search_that_hash/__main__.py

```diff
@@ -59,31 +61,30 @@ def main(**kwargs):
     \n
         sth --text "5f4dcc3b5aa765d61d8327deb882cf99"
     """
-    set_logger(kwargs)
-    logger.trace(kwargs)
+    
+    levels = {1:logging.WARNING,2:logging.INFO,3:logging.DEBUG}
+    try:
+        coloredlogs.install(level=levels[kwargs['verbose']])
+    except:
+        logger.propagate = False
```

Comment:
Why the try; except, add a comment to explain? At least `except <Error>` so we know the reason why the error is happening?

Reply:
Hey, this is for when no verbosity was specified so it removes it